dir.common.props

```diff
@@ -98,4 +78,25 @@
 
   </PropertyGroup>
 
+  <!-- Version numbers for binaries and packages
+       [ARCADE REMOVE] These properties can be removed once native binaries use Arcade versioning
+       They are included in common for packaging, which is onboarded to Arcade, but uses the version
+       numbers for native binaries for some outputs. This should be kept in sync or moved to eng/Versions.props -->
+  <PropertyGroup>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">3.0.0</PackageVersion>
+    <ProductVersion>$(PackageVersion)</ProductVersion>
+    <PreReleaseLabel>preview6</PreReleaseLabel>
+    <PreReleaseVersionLabel>$(PreReleaseLabel)</PreReleaseVersionLabel>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
+    <StableVersion Condition="'$(StabilizePackageVersion)' == 'true' and '$(StableVersion)' == ''">$(PackageVersion)</StableVersion>
+    <NativeMajorVersion>4</NativeMajorVersion>
+    <NativeMinorVersion>30</NativeMinorVersion>
+    <MajorVersion>$(NativeMajorVersion)</MajorVersion>
+    <MinorVersion>$(NativeMinorVersion)</MinorVersion>
+
+    <!-- This is necessary as we are using a mix of Arcade and BuildTools versioning.
+        Otherwise, local-dev packages won't produce long-named assets-->
+    <OfficialBuildId Condition="'$(OfficialBuildId)' == ''">19991212.0</OfficialBuildId>
```

Comment:
We really should not be setting this for non-official builds. This will make it so that all builds (including local dev and PR) will be treated as official. The non-official builds will always have the same build number and same name for the build version props file, which will not be re-created if it already exists.

I don't see why this would be necessary for local dev packages. For non-official builds, we'd use the default value of `BuildVersionFile` set by the BuildTools targets, and it would be created and then pulled in for packaging. The issue where the official build was not producing packages with the version in the name is due to the fact that we explicitly set `BuildVersionFile` for official builds in a props file that was not included by the packaging projects didn't.